code/datalab-app/src/app/components/welcome/DescribeElement.spec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import DescribeElement from './DescribeElement';
+
+function setup(props) {
```

Comment:
Why this pattern of pulling out the component construction when it is a one liner?